mobile/scenes/Home/components/CourseProgressBar.js

```diff
@@ -36,16 +34,6 @@ const mapStateToProps = (state) => {
 
 export default compose(
   connect(mapStateToProps),
-  graphql(currentLessonQuery, {
-    name: 'currentLesson',
-    skip: props => !props.selectedCourse,
-    options: (ownProps) => {
-      const selectedCourse = ownProps.selectedCourse._id
-      return ({
-        variables: { courseId: selectedCourse },
-        fetchPolicy: 'network-only'
-      })
-    }
-  }),
+  getGraphqlForCurrentLesson(graphql, 'currentLesson', props => !props.selectedCourse),
```

Comment:
This is hard to read, I'd prefer sending an object down like
{name: 'currentLesson', whateverThisIs: props => !props.selectedCourse}